packages/fast-layouts-react/src/grid/grid.props.ts

```diff
@@ -69,6 +74,12 @@ export interface GridHandledProps extends GridManagedClasses {
      * Use this property to support -ms-grid when Grid is a child of an element who's display property is set to -ms-grid
      */
     row?: number;
+
+    /**
+     * The CSS grid display, "grid" when true and "-ms-grid" when false
+     * Provide this prop when doing server side rendering
+     */
+    displayType?: GridDisplay;
```

Comment:
This seems a little misleading, as what it is is a CSS property name for the grid property. My preference would be:
`cssGridPropertyName`.

Reply:
I'm good with this.

Reply:
sounds good